packages/main/src/CheckBox.js

```diff
@@ -290,7 +290,9 @@ class CheckBox extends UI5Element {
 	syncLabel() {
 		this._label = { ...this._label };
 		this._label.text = this.text;
-		this._label.wrap = this.wrap;
+		/* temporary workaround. remove after all wrap properties in the relevant components are renamed to wrappingType */
+		this._label.wrappingType = this.wrap ? "Normal" : "None";
```

Comment:
Should we add a wrappingType prop to CheckBox?

Reply:
Hi @gkmv we should, but perhaps as part of a separate change

Reply:
Ok. I will create a follow-up change.